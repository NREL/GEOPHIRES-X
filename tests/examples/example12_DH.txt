----------------------------------------------------------------------------------------
GEOPHIRES v2.0 Input File
Geothermal Electricity Example Problem using Percentage Thermal Drawdown Model (Example 4)
Created by Charley Walton on 7/25/2022
Last modified on 8/12/2022

Description: This example considers Cornell's existing district heating network assesses
the performance of a proposed geothermal well. A known heat demand profile for the district
is processed using the district_heat.py module See Charley Walton's SULI Research Report
"Techno-Economic Simulations of Geothermal District Heating Systems with GEOPHIRES" for
more details on the case study.

Note: The GEOPHIRES v2.0 python code scans the input file for each necessary "parameter,
value". Therefore, the user cannot change the string before the parameter value and the
comma is the required delimeter. Each paramater string may only appear once in this
input file. However, the parameters can be in any order and comments can be added
either by including extra lines or by typing it after the parameter value, separated by
a comma (see examples below). If a necessary parameter is not provided, a default value
is assumed, and a warning message is printed to the console.
----------------------------------------------------------------------------------------

*** Subsurface technical parameters ***
****************************************
Reservoir Model,4,			--- Percentage thermal drawdown model
Drawdown Parameter,0.003, 		--- In [1/year].
Reservoir Depth,3.5, 			--- [km]
Number of Segments,1,			--- [-]
Gradient 1,29.5, 			--- [deg.C/km]
Number of Production Wells,2,		--- [-]
Number of Injection Wells,2,		--- [-]
Production Well Diameter,6,		--- [inch]
Injection Well Diameter,6,		--- [inch]
Ramey Production Wellbore Model,0,	--- Should be 0 (disable) or 1 (enable)
Production Wellbore Temperature Drop,3,	--- [deg.C]
Injection Wellbore Temperature Gain,0,	--- [deg.C]
Production Flow Rate per Well,50, 	--- [kg/s]
Maximum Temperature,400,		--- [deg.C]
Reservoir Volume Option,4,		--- Should be 1, 2, 3, or 4. See manual for details.
Reservoir Volume,1.25e8,		--- [m3] (required for reservoir volume option 3 and 4
Water Loss Fraction,0.0,		--- [-] (total geofluid lost)/(total geofluid produced)
Injectivity Index,10,			--- [kg/s/bar]
Productivity Index,10,			--- [kg/s/bar]
Injection Temperature,50,		--- [deg.C]
Maximum Drawdown,1,			--- [-] Maximum allowable drawdown before redrilling (a value of 0.1 means redrilling of 10% drawdown)
Reservoir Heat Capacity,1000,		--- [J/kg/K]
Reservoir Density,2700,			--- [kg/m3]
Reservoir Thermal Conductivity,3,	--- [W/m/K]


*** Surface technical parameters ***
************************************
End-Use Option,2, --- Direct use
Power Plant Type, 7,				---[-] District Heating
Circulation Pump Efficiency,0.8,	--- [-]
Utilization Factor,0.8,			--- [-]
Surface Temperature,12,			--- [deg.C]
Ambient Temperature,12,			--- [deg.C]
End-Use Efficiency Factor,0.8,		--- [-]

District Heating Demand Option,1,					--- Should be 1 or 2. See manual or below for option details
District Heating Demand File Name,Examples/cornell_heat_demand.csv,		--- hourly MW thermal demand in a CSV file
District Heating Demand Data Time Resolution,1,				--- 1 for hourly, 2 for daily
District Heating Demand Data Column Number,2,				---

Peaking Fuel Cost Rate,0.0273, 					--- [$/kWh] Cost of natural gas for peak boiler use
Peaking Boiler Efficiency,0.85,			`		--- Should be between 0 and 1, defaults to 0.85
District Heating Piping Cost Rate,1200,					--- [$/m] used for calculating surface piping capital cost for district heat
District Heating Road Length,3, 				---[km] supersedes model option 2 if any value is entered


*** Economic/Financial Parameters ***
*************************************
Plant Lifetime,30,						--- [years]
Economic Model,2,						--- Should be 1 (FCR model), 2 (Standard LCOE/LCOH model), or 3 (Bicycle model).
Discount Rate,0.05,
Inflation Rate During Construction,0,				--- [-]
Well Drilling and Completion Capital Cost Adjustment Factor,1,	--- [-] Use built-in well cost correlation as is
Well Drilling Cost Correlation,1,				--- [-] Use built-in well drilling cost correlation #1
Reservoir Stimulation Capital Cost,1,				--- [M$/injection well] Reservoir stimulation capital cost per injection well
Surface Plant Capital Cost Adjustment Factor,1,			--- [-] Use built-in surface plant cost correlation as is
Field Gathering System Capital Cost Adjustment Factor,1,	--- [-] Use built-in pipeline distribution cost correlation as is
Exploration Capital Cost Adjustment Factor,0,			--- [-] Use built-in exploration cost correlation as is
Wellfield O&M Cost Adjustment Factor,1,				--- [-] Use built-in wellfield O&M cost correlation as is
Water Cost Adjustment Factor,1,					--- [-] Use built-in water cost correlation as is
Surface Plant O&M Cost Adjustment Factor,1,			--- [-] Use built-in surface plant O&M cost correlation as is


*** Simulation Parameters ***
Print Output to Console,1,		--- [-] Should be 0 (don't print results to console) or 1 (print results to console)
Time steps per year,3,			--- [1/year]
