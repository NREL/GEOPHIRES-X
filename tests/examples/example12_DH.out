                               *****************
                               ***CASE REPORT***
                               *****************

Simulation Metadata
----------------------
 GEOPHIRES Version: 3.4.0
 GEOPHIRES Build Date: 2022-06-30
 Simulation Date: 2024-02-12
 Simulation Time:  09:55
 Calculation Time:      0.069 sec

                           ***SUMMARY OF RESULTS***

      End-Use Option: Direct-Use Heat
      Annual District Heating Demand:                       242.90 GWh/year
      Average Annual Geothermal Heat Production:            148.43 GWh/year
      Average Annual Peaking Fuel Heat Production:           94.46 GWh/year
      Average Direct-Use Heat Production:                    19.91 MW
      Direct-Use heat breakeven price:                        8.66 USD/MMBTU
      Number of production wells:                             2
      Number of injection wells:                              2
      Flowrate per production well:                          50.0 kg/sec
      Well depth (or total length, if not vertical):          3.5 kilometer
      Geothermal gradient:                                    0.0295 degC/m


                           ***ECONOMIC PARAMETERS***

      Economic Model = Standard Levelized Cost
      Interest Rate:                                          5.00
      Accrued financing during construction:                  0.00
      Project lifetime:                                      30 yr
      Capacity factor:                                       85.2 %
      Project NPV:                                         -17.18 MUSD
      Project IRR:                                           0.02 %
      Project VIR=PI=PIR:                                    0.63
      Project MOIC:                                          0.18

                          ***ENGINEERING PARAMETERS***

      Number of Production Wells:                             2
      Number of Injection Wells:                              2
      Well depth (or total length, if not vertical):          3.5 kilometer
      Water loss rate:                                        0.0
      Pump efficiency:                                       80.0
      Injection temperature:                                 50.0 degC
      User-provided production well temperature drop
      Constant production well temperature drop:              3.0 degC
      Flowrate per production well:                          50.0 kg/sec
      Injection well casing ID:                               6.000 in
      Production well casing ID:                              6.000 in
      Number of times redrilling:                             0


                         ***RESOURCE CHARACTERISTICS***

      Maximum reservoir temperature:                        400.0 degC
      Number of segments:                                     1
      Geothermal gradient:                                    0.0295 degC/m


                           ***RESERVOIR PARAMETERS***

      Reservoir Model = Annual Percentage Thermal Drawdown Model
      Annual Thermal Drawdown:                                0.300 1/year
      Bottom-hole temperature:                              115.25 degC
      Warning: the reservoir dimensions and thermo-physical properties
               listed below are default values if not provided by the user.
               They are only used for calculating remaining heat content.
      Reservoir volume provided as input
      Reservoir volume:                               125000000 m**3
      Reservoir hydrostatic pressure:                       100.00 kPa
      Plant outlet pressure:                                446.32 kPa
      Production wellhead pressure:                         515.27 kPa
      Productivity Index:                                    10.00 kg/sec/bar
      Injectivity Index:                                     10.00 kg/sec/bar
      Reservoir density:                                   2700.00 kg/m**3
      Reservoir heat capacity:                             1000.00 J/kg/kelvin


                           ***RESERVOIR SIMULATION RESULTS***

      Maximum Production Temperature:                       112.2 degC
      Average Production Temperature:                       109.3 degC
      Minimum Production Temperature:                       106.4 degC
      Initial Production Temperature:                       112.2 degC
      Average Reservoir Heat Extraction:                     24.88 MW
      Wellbore Heat Transmission Model = Constant Temperature Drop:       3.0 degC
      Average Injection Well Pump Pressure Drop:           1597.5 kPa
      Average Production Well Pump Pressure Drop:          1393.6 kPa


                          ***CAPITAL COSTS (M$)***

         Drilling and completion costs:                      27.30 MUSD
         Drilling and completion costs per well:              6.82 MUSD
         Stimulation costs:                                   1.00 MUSD
         Surface power plant costs:                          12.40 MUSD
            of which Peaking Boiler Cost:                     3.99 MUSD
         Field gathering system costs:                        2.51 MUSD
         District Heating System Cost:                        2.70 MUSD
         Total surface equipment costs:                      14.91 MUSD
         Exploration costs:                                   0.00 MUSD
      Total capital costs:                                   45.91 MUSD


                ***OPERATING AND MAINTENANCE COSTS (M$/yr)***

         Wellfield maintenance costs:                         0.48 MUSD/yr
         Power plant maintenance costs:                       0.74 MUSD/yr
         Water costs:                                         0.00 MUSD/yr
         Average Reservoir Pumping Cost:                      0.20 MUSD/yr
         Annual District Heating O&M Cost:                    0.37 MUSD/yr
         Average Annual Peaking Fuel Cost:                    3.03 MUSD/yr
      Total operating and maintenance costs:                  1.79 MUSD/yr


                           ***SURFACE EQUIPMENT SIMULATION RESULTS***

      Maximum Net Heat Production:                           20.89 MW
      Average Net Heat Production:                           19.91 MW
      Minimum Net Heat Production:                           18.92 MW
      Initial Net Heat Production:                           20.89 MW
      Average Annual Heat Production:                       146.84 GWh
      Annual District Heating Demand:                       242.90 GWh/year
      Maximum Daily District Heating Demand:               1683.32 MWh/day
      Average Daily District Heating Demand:                665.47 MWh/day
      Minimum Daily District Heating Demand:                216.86 MWh/day
      Maximum Geothermal Heating Production:                 20.87 MW
      Average Geothermal Heating Production:                 16.94 MW
      Minimum Geothermal Heating Production:                  9.04 MW
      Maximum Peaking Boiler Heat Production:                51.21 MW
      Average Peaking Boiler Heat Production:                10.78 MW
      Minimum Peaking Boiler Heat Production:                 0.00 MW
      Average Pumping Power:                                  0.39 MW

                            ************************************************************
                            *  HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                            ************************************************************
  YEAR         THERMAL              GEOFLUID               PUMP              GEOTHERMAL
               DRAWDOWN            TEMPERATURE             POWER            HEAT OUTPUT
                                    (deg C)                (MWe)               (MWt)
   0            1.0000               112.25                0.3761             20.8916
   1            0.9982               112.05                0.3768             20.8252
   2            0.9965               111.85                0.3774             20.7587
   3            0.9947               111.66                0.3780             20.6923
   4            0.9929               111.46                0.3787             20.6259
   5            0.9912               111.26                0.3793             20.5594
   6            0.9894               111.06                0.3800             20.4930
   7            0.9877               110.86                0.3806             20.4266
   8            0.9859               110.67                0.3812             20.3601
   9            0.9841               110.47                0.3819             20.2937
  10            0.9824               110.27                0.3825             20.2273
  11            0.9806               110.07                0.3831             20.1608
  12            0.9788               109.87                0.3838             20.0944
  13            0.9771               109.68                0.3844             20.0280
  14            0.9753               109.48                0.3850             19.9615
  15            0.9735               109.28                0.3857             19.8951
  16            0.9718               109.08                0.3863             19.8287
  17            0.9700               108.88                0.3869             19.7622
  18            0.9683               108.69                0.3876             19.6958
  19            0.9665               108.49                0.3882             19.6294
  20            0.9647               108.29                0.3888             19.5629
  21            0.9630               108.09                0.3894             19.4965
  22            0.9612               107.90                0.3901             19.4301
  23            0.9594               107.70                0.3907             19.3636
  24            0.9577               107.50                0.3913             19.2972
  25            0.9559               107.30                0.3919             19.2308
  26            0.9541               107.10                0.3925             19.1643
  27            0.9524               106.91                0.3932             19.0979
  28            0.9506               106.71                0.3938             19.0315
  29            0.9489               106.51                0.3944             18.9650


                              *******************************************************************
                              *  ANNUAL HEATING, COOLING AND/OR ELECTRICITY PRODUCTION PROFILE  *
                              *******************************************************************
  YEAR           GEOTHERMAL          PEAKING BOILER       RESERVOIR HEAT          RESERVOIR         PERCENTAGE OF
              HEATING PROVIDED      HEATING PROVIDED        EXTRACTED            HEAT CONTENT     TOTAL HEAT MINED
                 (GWh/year)            (GWh/year)           (GWh/year)            (10^15 J)              (%)
   1               153.1                  89.8                191.38                21.33                3.13
   2               152.8                  90.1                190.99                20.65                6.25
   3               152.5                  90.4                190.60                19.96                9.37
   4               152.2                  90.7                190.21                19.27               12.48
   5               151.8                  91.0                189.81                18.59               15.58
   6               151.5                  91.4                189.42                17.91               18.68
   7               151.2                  91.7                189.02                17.23               21.77
   8               150.9                  92.0                188.62                16.55               24.85
   9               150.6                  92.3                188.22                15.87               27.93
  10               150.3                  92.6                187.81                15.20               31.00
  11               149.9                  93.0                187.41                14.52               34.06
  12               149.6                  93.3                187.00                13.85               37.12
  13               149.3                  93.6                186.60                13.18               40.17
  14               149.0                  93.9                186.19                12.51               43.21
  15               148.6                  94.3                185.79                11.84               46.25
  16               148.3                  94.6                185.38                11.17               49.28
  17               148.0                  94.9                184.97                10.50               52.30
  18               147.6                  95.3                184.55                 9.84               55.32
  19               147.3                  95.6                184.14                 9.18               58.33
  20               147.0                  95.9                183.72                 8.52               61.33
  21               146.6                  96.3                183.31                 7.86               64.33
  22               146.3                  96.6                182.89                 7.20               67.32
  23               146.0                  96.9                182.48                 6.54               70.30
  24               145.6                  97.2                182.06                 5.88               73.28
  25               145.3                  97.6                181.64                 5.23               76.25
  26               145.0                  97.9                181.23                 4.58               79.21
  27               144.6                  98.2                180.81                 3.93               82.17
  28               144.3                  98.6                180.40                 3.28               85.12
  29               144.0                  98.9                179.98                 2.63               88.06
  30                95.8                  99.2                119.77                 2.20               90.02
